packages/moonstone/Input/pointer.js

```diff
@@ -0,0 +1,46 @@
+import {handle, preventDefault, returnsTrue, stop} from '@enact/core/handle';
+
+let active;
+let capturedDown = false;
+
+// if there is a focused input (there should be) and the target is not the input
+const shouldCapture = ({target}) => active && target !== active;
+const isCapturing = () => capturedDown && active;
+const setCapturing = (capturing) => returnsTrue(() => (capturedDown = capturing));
+
+const handlePointerDown = handle(
+	shouldCapture,				// If we should capture the click
+	preventDefault,				// prevent the down event bubbling
+	stop,						// (and stop propagation to support touch)
+	setCapturing(true)			// and flag that we've started capturing a down event
+);
+
+const handlePointerClick = handle(
+	isCapturing,				// if a down event was captured
+	stop,						// prevent the click event from propagating
+	setCapturing(false),		// clear the capturing flag
+	() => active.blur()			// and blur the active node
+);
+
+// Lock the pointer from emitting click events until released
+const lockPointer = (target) => {
+	active = target;
+	document.addEventListener('mousedown', handlePointerDown, {capture: true});
+	document.addEventListener('touchstart', handlePointerDown, {capture: true});
```

Comment:
Since we're now preventing pointer/mouse events when focusing an `Input`, I feel it may make sense to prevent `mouseup`/`touchend` events as well. It might feel inconsistent that those events can still be fired on background elements while their originating events (`mousedown`/`touchstart`) are prevented. Thoughts?